emmet-api/app.py

```diff
@@ -1,4 +1,3 @@
-import ddtrace.auto
```

Comment:
please make sure to keep this import (feel free to add a noqa comment for linters)